egs/fisher_callhome_spanish/st1/local/score_bleu.sh

```diff
@@ -13,7 +13,6 @@ bpemodel=""
 filter=""
 case=lc
 set=""
-single_ref=false  # If true, evaluate with a single reference
```

Comment:
My understanding is to simplify the recipe by always using multiple references, right?

Reply:
both 4-ref and a single ref